projects/objects/backgrounds/protos/TexturedBackgroundLight.proto

```diff
@@ -102,16 +102,20 @@ PROTO TexturedBackgroundLight [
     else
       io.stderr:write("Unknown texture identifier\n")
     end
+    if context.coordinate_system == "ENU" then
+      direction = {direction[3], direction[1], direction[2]}
+    end
+
   }%
   DirectionalLight {
     ambientIntensity 1
     direction %{=direction[1]}% %{=direction[2]}% %{=direction[3]}%
     color %{=color[1]}% %{=color[2]}% %{=color[3]}%
     intensity %{=luminosity * intensity}%
     %{ if on then }%
-      on TRUE
+    on TRUE
     %{ else }%
-      on FALSE
+    on FALSE
     %{ end }%
```

Comment:
Just for the note (I know this PR is not ready, I just saw this), most of the PROTOs are preferring 'template' indentation rather than 'resulting' indentation (but not all of them, we are not 100% coherent with this).
Changing this to 'resulting' indentation is fine for me, but since there might be many changes (due to indentation) it is probably better to do it in a preliminary PR to keep the diff of this PR as readable as possible (since it is already going to be huge).